apps/wallet-dashboard/app/(protected)/vesting/page.tsx

```diff
@@ -221,6 +220,12 @@ export default function VestingDashboardPage(): JSX.Element {
         );
     }
 
+    const isSupplyIncreaseVestingScheduleEmpty =
+        !supplyIncreaseVestingSchedule.totalVested &&
+        !supplyIncreaseVestingSchedule.totalLocked &&
+        !supplyIncreaseVestingSchedule.availableClaiming &&
+        !supplyIncreaseVestingSchedule.totalStaked &&
+        !supplyIncreaseVestingSchedule.totalEarned;
```

Comment:
we can migrate this logic to `useGetSupplyIncreaseVestingObjects` hook, and use it in SupplyIncreaseVestingOverview too